api/client/network/create.go

```diff
@@ -55,6 +56,7 @@ func newCreateCommand(dockerCli *client.DockerCli) *cobra.Command {
 	flags.StringSliceVar(&opts.labels, "label", []string{}, "Set metadata on a network")
 	flags.BoolVar(&opts.internal, "internal", false, "Restrict external access to the network")
 	flags.BoolVar(&opts.ipv6, "ipv6", false, "Enable IPv6 networking")
+	flags.BoolVar(&opts.manuallyAttachable, "manually-attachable", false, "Enable manual container attachment")
```

Comment:
@cpuguy83 one cannot run services from a worker node. Its always launched from the manager. Thats the swarmkit's security model. Also `docker run --net=container:<task>` isnt going to allocate any resources from the manager. It is just sharing the namespace.